crashreports/internal/win32/CrashReportContext.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <string>
+#include <map>
+
+class CrashReportContext
+{
+	static constexpr int MaxBufferLength{ 2048 };
+	static constexpr int MaxCommandLength{ 8192 };
+
+	wchar_t senderPath[MaxBufferLength]{};
+	wchar_t reportURL[MaxBufferLength]{};
+	wchar_t parameters[MaxBufferLength]{};
+	wchar_t command[MaxCommandLength]{};
```

Comment:
What is the reason to use fixed-size strings?

Reply:
to avoid using heap allocations after exception occurs

Reply:
Also because it is statically constructed?  Yet it is a static constructed in a function, not file scope, and that function is done when already starting up the application, somewhat late in the startup process.

Reply:
@vsverchinsky but it's constructed before the exception.